src/java/htsjdk/samtools/cram/build/CramNormalizer.java

```diff
@@ -123,9 +124,15 @@ public void normalize(final ArrayList<CramCompressionRecord> records,
 
             if (record.isUnknownBases()) {
                 record.readBases = SAMRecord.NULL_SEQUENCE;
-            } else
+            } else {
+                if (refBases != null) {
                 record.readBases = restoreReadBases(record, refBases, refOffset_zeroBased,
                         substitutionMatrix);
+                } else {
+                     throw new CRAMException(String.format("Contig %s not found in the reference file.", header.getSequence(record.sequenceId).toString()));
+                }
+            }
```

Comment:
why not just `else if { ...} else { ... }` instead of nesting an `if/else` inside an `else`?